tests/unit/_bootstrap.php

```diff
@@ -1,182 +1,98 @@
-      array(
-        'label' => 'Nickname',
-      ),
-      'position' => '2',
-    ),
-    array(
-      'type' => 'text',
-      'name' => 'Age',
-      'id' => '2',
-      'unique' => '1',
-      'static' => '0',
-      'params' =>
-      array(
-        'required' => '',
-        'validate' => 'number',
-        'label' => 'Age',
-      ),
-      'position' => '3',
-    ),
-    array (
-      'type' => 'divider',
-      'name' => 'Divider',
-      'id' => 'divider',
-      'unique' => '0',
-      'static' => '0',
-      'params' => '',
-      'position' => '4',
-    ),
-    array (
-      'type' => 'radio',
-      'name' => '3-way choice',
-      'id' => '3',
-      'unique' => '1',
-      'static' => '0',
-      'params' =>
-      array (
-        'values' =>
-        array (
-          0 =>
-          array (
-            'value' => '1',
-          ),
-          1 =>
-          array (
-            'value' => '2',
-          ),
-          2 =>
-          array (
-            'value' => '3',
-          ),
-        ),
-        'required' => '',
-        'label' => '3-way choice',
-      ),
-      'position' => '5',
-    ),
-    array (
-      'type' => 'html',
-      'name' => 'Custom text or HTML',
-      'id' => 'html',
-      'unique' => '0',
-      'static' => '0',
-      'params' =>
-      array (
-        'text' => 'Subscribe to our newsletter and join [mailpoet_subscribers_count] other subscribers.',
-      ),
-      'position' => '6',
-    ),
-    array(
-      'type' => 'text',
-      'name' => 'Email',
-      'id' => 'email',
-      'unique' => '0',
-      'static' => '1',
-      'params' =>
-      array(
-        'label' => 'Email',
-        'required' => 'true',
-      ),
-      'position' => '7',
-    ),
-    array(
-      'type' => 'submit',
-      'name' => 'Submit',
-      'id' => 'submit',
-      'unique' => '0',
-      'static' => '1',
-      'params' =>
-      array(
-        'label' => 'Subscribe!',
-      ),
-      'position' => '8',
-    ),
-  )
-);
-
-/**
- * Simple class mocking dynamic segment filter.
- */
-class DynamicSegmentFilter {
-  protected $ids;
-
-  function __construct($ids) {
-    $this->ids = $ids;
-  }
-  
-  public function toSql($orm) {
-    return $orm->whereIn(Subscriber::$_table . '.id', $this->ids);
-  }
-}
\ No newline at end of file
+<?php
+
+$console = new \Codeception\Lib\Console\Output([]);
+
+$cacheDir = '/tmp';
+if(is_dir(getenv('WP_TEST_CACHE_PATH'))) {
+  $cacheDir = getenv('WP_TEST_CACHE_PATH');
+}
+
+$console->writeln('Clearing AspectMock cache...');
+exec('rm -rf ' . $cacheDir . '/_transformation.cache');
+
+$console->writeln('Initializing AspectMock library...');
+$kernel = \AspectMock\Kernel::getInstance();
+$kernel->init(
+  array(
+    'debug' => true,
+    'appDir' => __DIR__ . '/../../',
+    'cacheDir' => $cacheDir,
+    'includePaths' => [__DIR__ . '/../../lib']
+  )
+);
+
+abstract class MailPoetTest extends \Codeception\TestCase\Test {
+  protected $backupGlobals = true;

```

Comment:
These settings were introduced so the testing environment won't mess with WordPress core workings that rely on global variables. Since we don't load WP for unit tests we can try to get rid of these, but we should keep them in mind in case we get some weird issues.